demo/app/Prescriptor.py

```diff
@@ -3,10 +3,42 @@
 import pandas as pd
 import numpy as np
 from keras.models import load_model
-from unileaf_util.framework.transformers.data_encoder import DataEncoder
+from sklearn.preprocessing import MinMaxScaler
 
 from . import constants
 
+class Encoder:
```

Comment:
Super simple encoder based off unileaf-util. The only real complexity in the DataEncoder class is dealing with different data types and categorical variables. Since these are all continuous float/integers we do the most basic case where you simply use an SkLearn MinMaxScaler.